docs/sources/flow/reference/components/discovery.kubernetes.md

```diff
@@ -50,14 +36,6 @@ The `role` argument is required to specify what type of targets to discover.
 `role` must be one of `node`, `pod`, `service`, `endpoints`, `endpointslice`,
 or `ingress`.
 
-The following sub-blocks are supported:
-
-Name | Description | Required
----- | ----------- | --------
-[`namespaces`](#namespaces-block) | Information about which Kubernetes namespaces to search. | no
-[`selectors`](#selectors-block) | Selectors to limit objects selected. | no
-[`http_client_config`](#http_client_config-block) | HTTP client configuration for Kubernetes requests. | no
-
 ### `node` role
```

Comment:
Not explicitly related to the PR but should `node` become node to match the change in blocks below?

Reply:
Applies to all the other roles also

Reply:
Yep, just did that 👍